### ISSUE TITLE
Cross validation may be better

### ISSUE THREAD

**Issue by yianzhongguo** [JohnleeHIT/Brats2019#9]
Hi sir, your code is very good. It looks like that you use the first 50 patients in training data set to validate the outcome of the model after many epoch training. I think maybe cross validation can be used, do you agree, thank you very much!

**Comment by JohnleeHIT** [comment]
Yes, you are right, i did five fold cross validation when finnally submit the result.
